src/pages/authentication/LoginPage.tsx

```diff
@@ -0,0 +1,56 @@
+import { useTranslation } from "react-i18next";
+import {
+    StyledContent,
+    StyledLayout,
+} from "../../components/StyledComponents";
+import {InputField} from "../../components/form/InputField";
+import {ErrorMessage,  Form, Formik, } from "formik";
```

Comment:
It would be great to use antd (https://ant.design/components/) 
not high priority for the PoC, but will integrate into the current design,